src/app/shared/store/user.state.ts

```diff
@@ -330,8 +331,11 @@ export class UserState {
 
   @Action(OnCreateApplicationFail)
   onCreateApplicationFail({ dispatch }: StateContext<UserStateModel>, { payload }: OnCreateApplicationFail): void {
-    throwError(payload);
-    dispatch(new ShowMessageBar({ message: 'На жаль виникла помилка', type: 'error' }));
+    throwError(payload);        
+    dispatch(new ShowMessageBar({ message: payload.error.status === 429 
+      ? `Ліміт заяв перевищено, повторіть спробу через 6 днів` 
```

Comment:
display actual time as discussed

Reply:
Need some tip how to get "Retry-After" in user.state.ts

Reply:
https://stackoverflow.com/questions/36098913/convert-seconds-to-days-hours-minutes-and-seconds

Reply:
should I get the Retry-After from payload or add  some another parameter?

Reply:
payload.headers.get('retry-after') - is it correct way to get seconds for next attempt? as I get now null and headers in payload looks like: 
headers: Map(2)
[[Entries]]
0: {"content-length" => Array(1)}
1: {"content-type" => Array(1)}
size: 2